pages.ko/common/zola.md

```diff
@@ -0,0 +1,24 @@
+# zola
+
+> 단일 실행 파일에 모든 기능이 들어있는 정적 사이트 생성기.
+> 더 많은 정보: <https://www.getzola.org/documentation/getting-started/cli-usage/>.
+
+- 주어진 디렉토리에 Zola가 사용하는 디렉토리 구조를 생성:
+
+`zola init {{my_site}}`
+
+- `public` 디렉토리에 전체 사이트를 빌드 (이미 존재하는 디렉토리가 있다면 삭제):
+
+`zola build`
+
+- 별도의 디렉토리에 전체 사이트를 빌드:
+
+`zola build --output-dir {{path/to/output_directory/}}`
```

Comment:
Thanks for your contribution, can you translate this path placeholder too?

Reply:
Sure thing! I've also translated the word `my_site` into more Korean-specific term.